fiftyone/operators/operations.py

```
@@ -719,3 +747,49 @@ def browser_download(self, url, filename=None):
+    "thumbsup",
+    "thumbsdown",
+    "pin",
+    "star",
+    "x",
+    "bookmark",
+}
+
+_VALID_SELECTION_TYPES = {"default", "alt"}
+
+
+def _validate_meta(samples, meta):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_validate_meta`

Add return type annotation: `None`

(ANN202)